settings.py

```diff
@@ -25,7 +25,7 @@
 LOGGING['root']['level'] = logging.DEBUG
 for logger in list(LOGGING['loggers'].keys()):
     if logger not in ['filtercascade', 'mohawk.util', 'post_request_task']:
-        del LOGGING['loggers'][logger]
+        LOGGING['loggers'][logger]['level'] = logging.DEBUG
```

Comment:
This seems safer as we'll have the exact same logger defined in local dev and prod.